.github/workflows/release.yaml

```diff
@@ -48,6 +59,19 @@ jobs:
           docker push ghcr.io/scalar-labs/scalardb-server:${{ steps.version.outputs.version }}
           docker push ghcr.io/scalar-labs/scalardb-schema-loader:${{ steps.version.outputs.version }}
 
+      - name: Push containers to Azure Container Registry
```

Comment:
In this step, I create the following two images and push them to Azure Container Registry.

* `scalardb-server:X.Y.Z`
* `scalardb-server:X.Y`